src/lib.rs

```diff
@@ -42,6 +42,9 @@
 #![deny(unused_mut)]
 #![deny(missing_docs)]
 
+extern crate failure;
```

Comment:
Is this line required? If not we must remove it

Reply:
Yes, `failure` is the parent crate, required to use `failure_derive`.